api/HTMLElement.json

```diff
@@ -832,10 +832,10 @@
               "version_removed": "79"
             },
             "firefox": {
-              "version_added": "1"
+              "version_added": false
```

Comment:
Indeed, no `dropzone` in Gecko, it's been commented out since it was added in https://github.com/mozilla/gecko-dev/commit/f19ca9b71f06760fa782f23e2854119da2d5abe9